.github/workflows/main.yml

```diff
@@ -423,7 +423,7 @@ jobs:
           -exec sh -c 'for n; do echo "$n" | tee -a results.txt && python3 "$n" >> results.txt || exit 1; done' sh {} +
         env:
           # The ignore setting here is due to a new warning introduced in Matplotlib==3.6.0
-          PYTHONWARNINGS: "error,ignore:warn_name_set_on_empty_Forward::pyparsing"
+          PYTHONWARNINGS: "error,ignore:warn_name_set_on_empty_Forward::pyparsing, ignore::DeprecationWarning"
```

Comment:
Can you make this more specific, rather than ignoring all deprecation warnings? Ideally, we would want to see deprecations from calls made directly by Cantera, just not ones that occur indirectly through another package. While that may not be possible, we can at least limit the scope of this to be whatever package is currently issuing such warnings.

Reply:
Thanks @speth! I was able to make the ignore more specific in [880de60](https://github.com/Cantera/cantera/pull/1648/commits/880de60b167977485684c2c5ed32ce9aa086926d), and cherry-picked you CI fix.